lib/Doctrine/Migrations/Finder/MigrationFinder.php

```diff
@@ -4,6 +4,9 @@
 
 namespace Doctrine\Migrations\Finder;
 
+/**
+ * @internal
```

Comment:
Aren't these meant to be an extension point of the library ? Otherwise, how would people choose a different finder ?

Reply:
Yes, this shouldn't be internal. Changing it.

Reply:
implementations should not be internal either. Otherwise, the only way to use the extension point is to write your own implementation from scratch, even if you want the same behavior than one of the built-in implementations (as you cannot use them due to being internal and so not covered by BC).